src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ConnectionControl.xaml.cs

```
@@ -87,13 +78,17 @@ public bool UpdateConfigFromSelections(ConfigurationModel configuration)
                 if (issueConfigurationControl.CanSave)
                 {
                     string serializedConfigs = configuration.IssueReporterSerializedConfigs;
-                    Dictionary<Guid, string> configs = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigs);
-                    configs = configs ?? new Dictionary<Guid, string>();
 
-                    if (serializedConfigs != null)
+                    Dictionary<Guid, string> configs = new Dictionary<Guid, string>();
+                    try
                     {
                         configs = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigs);
                     }
+                    catch
+                    {
+                        MessageDialog.Show(Properties.Resources.InvalidConfiguration);
```

>MessageDialog [](start = 24, length = 13)

Should we report the Exception that we just caught? (e.ReportException())